phpseclib/phpseclib/CVE-2023-27560.yaml

```diff
@@ -0,0 +1,8 @@
+title:     Infinite Loop vulnerability
+link:      https://github.com/phpseclib/phpseclib/releases/tag/3.0.19
```

Comment:
Think this should go to https://github.com/advisories/GHSA-hm7p-r324-hhf3 ?

Reply:
Updated